components/augeas/Makefile

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
Please, move component to some category  - http://wiki.openindiana.org/oi/Categories+layout